server/testserver.go

```diff
@@ -28,8 +28,19 @@ import (
 	"github.com/cockroachdb/cockroach/storage/engine"
 	"github.com/cockroachdb/cockroach/util"
 	"github.com/cockroachdb/cockroach/util/hlc"
+	"github.com/cockroachdb/cockroach/util/log"
 )
 
+// StartTestServer starts a in-memory test server.
+func StartTestServer() *TestServer {
```

Comment:
I think it would be better here to accept a `t testing.TB` parameter and invoke t.Fatalf on failures. I think test code which is exported from a package should always explicitly take a the `testing.TB` for clarity and to avoid any misusage.